keps/sig-storage/4876-mutable-csinode-allocatable/README.md

```diff
@@ -650,6 +733,12 @@ Describe the metrics themselves and the reasons why they weren't added (e.g., co
 implementation difficulties, etc.).
 -->
 
+While the following metrics could provide additional visibility into the feature's operation, they weren't added because API server health metrics already indirectly measure the success of CSINode updates - if the API server is healthy, we expect updates to succeed:
```

Comment:
Indicate which precise apiserver metrics to check please. Most admins aren't likely to know them right off.